res/values-es/strings.xml

```diff
@@ -755,7 +755,7 @@
     <string name="Transparency">"Transparencia"</string>
     <string name="upload_info">"Se puede lograr una mejor calidad sin transparencia o con imágenes simples."</string>
     <string name="Compression_Quality_">"Calidad de compresión:"</string>
-    <string name="inapproriate_info">"Skins que incluyan cualquier cosa que implique odio, gore, obscenidad, violencia, drogas, sexo, desnudos, selfies de cuerpo entero y skins de Nebulous sin modificar."</string>
+    <string name="inappropriate_info">"Skins que incluyan cualquier cosa que implique odio, gore, obscenidad, violencia, drogas, sexo, desnudos, selfies de cuerpo entero y skins de Nebulous sin modificar."</string>
```

Comment:
I reviewed the `res/values` (english file) and it seems to contain a typo though. Even if it's just a correction, you must not change the original string name, because it ain't gonna work as it should (the translated text may keep in english if changed in any way). However, you made a nice catch. Please return the original name here, so the translation would work.

Reply:
Got it